app/src/main/res/layout/search_suggestions_hint.xml

```diff
@@ -89,7 +89,6 @@
             android:id="@+id/dismiss"
             android:layout_width="0dp"
             android:layout_height="48dp"
-            android:fontFamily="@font/metropolis_semibold"
```

Comment:
Same questions as above regarding whether or not we should add the font weight

Reply:
Same as others: this already had bold style applied, and it seemed to match better with the layout. So I just removed the font here.